appengine/mailgun/README.md

```diff
@@ -0,0 +1,59 @@
+## Mailgun on Google App Engine with Node.js
+
+> [Mailgun](https://www.mailgun.com/): The Email Service For Developers
+
+This example uses [Express.js](http://expressjs.com) and
+[node-mailgun](http://github.com/shz/node-mailgun).
+
+### Sign up for Mailgun
+
+1. Sign up for a [Mailgun account](https://mailgun.com/signup).
+1. Add a [new domain](https://mailgun.com/app/domains).
+1. Find your API key in your new domain's settings.
+
+### Configure
+
+Add these lines to the `app.yaml` file in the root of your application:
```

Comment:
I just realized there isn't a sample app.yaml in this PR, please add a sample app.yaml. They should only have to put in their api key, they shouldn't have to create the whole thing from scratch.